nimare/meta/utils.py

```diff
@@ -130,19 +130,22 @@ def _convolve_sphere(kernel, peaks):
         )
 
         all_spheres = all_spheres[idx, :]
+
         if sum_overlap:
             counts = counts[idx]
```

Comment:
I think this can be merged with (or removed in favor of) the `if sum_overlap:` you have below.